content/wiki/canonical_lints.md

```diff
@@ -59,6 +59,7 @@ clippy.semicolon_if_nothing_returned = "warn"
 clippy.shadow_unrelated = "warn"
 clippy.should_panic_without_expect = "warn"
 clippy.todo = "warn"
+clippy.too_many_arguments = "allow"
```

Comment:
I don't think this should be in-between all the allows. I'd probably put it above them?

Reply:
Alright I movied it, like that?